homework/week-2/newStyle.css

```diff
@@ -0,0 +1,100 @@
+.header {
+  background-color: #176ab0;
+  color: white;
+  text-align: center;
+  padding: 5px;
+}
+
+#main-content {
+  width: 350px;
+  float: left;
+  padding: 10px;
+}
+
+#footer {
+  background-color: #176ab0;
+  color: white;
+  clear: both;
+  text-align: center;
+  padding: 5px;
+}
+
+.badge {
+  background: grey;
+  float: right;
+  color: white;
+  width: 18px;
+  height: 18px;
+  text-align: center;
+  line-height: 18px;
+  border-radius: 50%;
+  padding: 10px;
+}
+
+.flex-wrapper {
+  font-family: Abel;
+  background-color: grey;
+  text-align: left;
+  padding-top: 25px;
+  padding-bottom: 25px;
+  font-size: 20px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  background-color: #2E2C2F;
+  color: #444;
+}
+
+#contributorsDiv.li {
+  text-align: center;
+  padding: 10px;
+  margin: auto;
+  display: inline;
+}
+
+#contributorsDiv.IMG {
+  padding: 20px;
+  margin: 20px;
+}
+
+#repo {
+  background-color: whitesmoke;
+  border-color: #2E2C2F;
+  width: 40%;
+  border: 5px;
+  padding: 20px;
+  margin: 20px;
+}
+
+#contribDiv {
+  background-color: whitesmoke;
+  border-color: #2E2C2F;
+  width: 40%;
+  border: 5px;
+  padding: 20px;
+  margin: 20px;
+}
+
+/* #contributionNumber {
```

Comment:
Don't leave commented out code, remove it instead! (Easier to read and change later)